builders/interact/ethereum-api/libraries/web3py.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Send Transactions & Deploy Contracts with Web3.py
```

Comment:
This ends up being "Send Transactions & Deploy Contracts with Web3.py | Tannsi Docs", which makes the title too long. It needs to be 60 characters or less

Reply:
I did `EVM Transactions & Contracts with Web3.py`